packages/instrumentation-pinecone/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@traceloop/instrumentation-pinecone",
+  "version": "0.1.0",
+  "description": "OpenTelemetry instrumentation for pinecone vector DB",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "repository": "traceloop/openllmetry-js",
+  "scripts": {
+    "build": "tsc --build tsconfig.json",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nodejs",
+    "tracing",
+    "attributes",
+    "semantic conventions"
+  ],
+  "author": "Traceloop",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14"
+  },
+  "files": [
+    "dist/src/**/*.js",
+    "dist/src/**/*.js.map",
+    "dist/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md",
+    "package.json"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@opentelemetry/core": "^1.8.0",
+    "@opentelemetry/instrumentation": "^0.44.0",
+    "@opentelemetry/semantic-conventions": "^1.0.0",
+    "@traceloop/ai-semantic-conventions": "^0.1.0"
+  },
+  "devDependencies": {
+    "@pinecone-database/pinecone": "^2.0.1"
+  },
+  "homepage": "https://github.com/traceloop/openllmetry-js/tree/main/packages/instrumentation-pinecone",
+  "gitHead": "ef1e70d6037f7b5c061056ef2be16e3f55f02ed5"
```

Comment:
How is this value determined and updated?

Reply:
Don't worry about it, it's auto-generated by `lerna` when we publish to `npm`